games-roguelike/zangband/zangband-2.7.4c.ebuild

```diff
@@ -25,39 +25,41 @@ DEPEND="${RDEPEND}
 
 S=${WORKDIR}/${PN}
 
+PATCHES=( "${FILESDIR}"/${P}-tk85.patch
+	"${FILESDIR}"/${P}-rng.patch
+	"${FILESDIR}"/${P}-tinfo.patch
+	"${FILESDIR}"/${P}-configure.patch
+	"${FILESDIR}"/${P}-makefile.patch )
+
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-tk85.patch \
-		"${FILESDIR}"/${P}-rng.patch \
-		"${FILESDIR}"/${P}-tinfo.patch \
-		"${FILESDIR}"/${P}-configure.patch \
-		"${FILESDIR}"/${P}-makefile.patch
+	default
+
 	mv configure.in configure.ac || die
 	eautoreconf
 }
 
 src_configure() {
-	egamesconf \
-		--datadir="${GAMES_DATADIR_BASE}" \
-		--with-setgid="${GAMES_GROUP}" \
-		--without-gtk \
+	local myconf=(
+		--libdir="${EPREFIX}"/$(get_libdir)/${PN}
+		--with-setgid="nobody"
+		--without-gtk
 		$(use_with tk tcltk)
+	)
+
+	econf "${myconf[@]}"
 }
 
-src_install() {
-	# Keep some important dirs we want to chmod later
-	keepdir "${GAMES_DATADIR}"/${PN}/lib/{apex,user,save,bone,info,xtra/help,xtra/music}
+DOCS=( readme
```

Comment:
all in one line or new line after/before ()

Reply:
you mean the preference is:

```sh
src_install()
{
    # content
}
```

? I personally prefer the opening bracket on the same line from a stylistic standpoint, is there a strong reason why it has to start on its own newline?

Reply:
Sorry, I was referring to preferred array syntax:
DOCS=( )
or
DOCS=(

)

Reply:
Ah hah, closing paren on its own line.

Yea I've actually seen it both ways so wasn't quite sure i there was a preference.

Perhaps a "style guide" *would* be helpful, as you've suggested before...

Reply:
> Perhaps a "style guide" _would_ be helpful, as you've suggested before...

There's a basic proxy-maint [Style Guide](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide) but there's plenty it doesn't cover. Not like majority of style issues are enforced (outside the few things in the policy guide), and maintainers may have their own preferences. Mostly comes down to the most common usage so it's less all over the place.